rocAL/rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -2098,13 +2126,13 @@ rocalVideoFileSourceSingleShard(
             decoder_type = DecoderType::FFMPEG_SOFTWARE_DECODE;
         auto [color_format, num_of_planes] = convert_color_format(rocal_color_format);
         auto decoder_mode = convert_decoder_mode(rocal_decode_device);
-        std::vector<size_t> dims = {context->user_batch_size(), sequence_length, video_prop.height, 
-                                    video_prop.width, static_cast<unsigned>(num_of_planes)};
+        auto [tensor_format, dims] = convert_color_format_to_sequence_layout(rocal_color_format, context->user_batch_size(),
+                                                                             video_prop.height, video_prop.width, num_of_planes, sequence_length);
         auto info  = TensorInfo(std::move(dims),
                                      context->master_graph->mem_type(),
                                      RocalTensorDataType::UINT8);
         info.set_color_format(color_format);
-        info.set_tensor_layout(RocalTensorlayout::NFHWC);
+        info.set_tensor_layout(tensor_format);
```

Comment:
Can we rename the variable tensor_format to layout_format/tensor_layout?

Reply:
Done